x-pack/plugins/security_solution/public/cases/components/configure_cases/index.tsx

```diff
@@ -42,6 +42,9 @@ const FormWrapper = styled.div`
 
     padding-top: ${theme.eui.paddingSizes.xl};
     padding-bottom: ${theme.eui.paddingSizes.xl};
+    .euiFlyout {
+      z-index: ${theme.eui.euiZNavigation + 1};
```

Comment:
@kevinlog FYI
We recently made a similar change in the Management area, but I think we hard-coded the zIndex value. I think this approach here (to use the values provided by `theme.eui.euiZNavigation` is better, so when we do refactor our implementation _(we wanted to make the change so that any `euiFlyout` in management automatically is set correctly)_ we use this instead 🙂